engine/table/src/main/java/io/deephaven/engine/table/impl/QueryTable.java

```diff
@@ -1493,16 +1493,15 @@ this, mode, columns, rowSet, getModifiedColumnSetForUpdates(), publishTheseSourc
 
                     final QueryTable resultTable;
                     final LivenessScope liveResultCapture = isRefreshing() ? new LivenessScope() : null;
-                    try (final SafeCloseable ignored1 = liveResultCapture != null ? liveResultCapture::release : null;
-                            final SafeCloseable ignored2 =
-                                    ExecutionContext.getDefaultContext().withUpdateGraph(getUpdateGraph()).open()) {
-                        // we open the default context here to ensure that the update processing happens in the default
-                        // context whether it is processed in parallel or not
+                    try (final SafeCloseable ignored1 = liveResultCapture != null ? liveResultCapture::release : null) {
                         try (final RowSet emptyRowSet = RowSetFactory.empty();
                                 final SelectAndViewAnalyzer.UpdateHelper updateHelper =
                                         new SelectAndViewAnalyzer.UpdateHelper(emptyRowSet, fakeUpdate)) {
 
-                            try {
+                            try (final SafeCloseable ignored2 =
```

Comment:
I don't understand why moving this helps at all. I thought the issue was with what was captured. Let's convene about this tomorrow.